modules/contact/doc/content/documentation/systems/Problem/ReferenceResidualProblem.md

```diff
@@ -50,6 +50,14 @@ and the temperature `saved` variable is applied in the heat conduction kernel
 
 !listing modules/combined/test/tests/reference_residual/reference_residual.i start=Kernels end=Functions
 
+`ReferenceResidualProblem` may also be used for generalized plane strain problems.  In this case, a reference scalar variable is needed to correspond with the scalar strain variable.
```

Comment:
On line 1 above, would you add spaces between each of the words in the class name please?